src/legacy_apps.ts

```diff
@@ -120,3 +120,7 @@ export function newUniqueApp(transport: any) {
 export function newBifrostKusamaApp(transport: any) {
   return newSubstrateApp(transport, 'BifrostKusama')
 }
+
+export function newVTBApp(transport: any) {
```

Comment:
This is not necessary. Adding the entry in supported_apps is enough

Reply:
Code removed